collector/processes_linux.go

```diff
@@ -118,8 +118,8 @@ func (c *processCollector) getAllocatedThreads() (int, map[string]int32, int, er
 				level.Debug(c.logger).Log("msg", "file not found when retrieving stats for pid", "pid", pid, "err", err)
 				continue
 			}
-			level.Debug(c.logger).Log("msg", "error reading stat for pid", "pid", pid, "err", err)
-			return 0, nil, 0, err
+			level.Debug(c.logger).Log("msg", "error reading stat for pid", "pid", pid.PID, "err", err)
```

Comment:
Perhaps we can drop this debug line, as the return of this func is an error anyway.